cmd/rootMain.go

```diff
@@ -43,6 +43,34 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 	// Configure logrus to emit simple text
 	log.SetFormatter(&log.TextFormatter{DisableTimestamp: true})
 
+	var execfile string
```

Comment:
Will this cause issues with `verify-install`?

Reply:
Still working that out.

Reply:
I'm trying to avoid creating temp directories before we fork, but don't fork for verify install. The only thing that's changed is that `plugin.IsInteractive` will now be false instead of true when running `--verify-install`.

Reply:
We shouldn't have any interactive behavior during `--verify-install`, so I think that's fine.

Reply:
Yeah, that's fine. May want to update the comment on Line 51 though.

Reply:
Done.